### ISSUE TITLE
plz tell me the reason why my model output is always be "i dont know"

### ISSUE THREAD

**Issue by yy2lyx** [jfriedson/Seq2Seq-Chatbot#1]

**Comment by jfriedson** [comment]
Some of the responses within the dialogue of the learning and validation data may contain the phrase "i dont know".  The optimizer of the neural network is trying to reduce the loss metric of the AI's responses; because so many responses in the data are "i dont know", the bot can get away with this response all of the time because it found a minima within the loss function for the data provided.

One dancing-around-the-issue workaround would be to strip the training data of all responses with the phrase "i dont know", however, the bot may find another phrase to trick the optimizer.  Additional improvements to mitigate the laziness of the neural network would be to find optimal hyperparameters through trial and error, trying out different optimizers, and optimizing different metrics, such as accuracy.

I don't have the resources for all of the testing required in this moment, but I will revisit this project in the future. The issue will remain open for now.
